bin/millau/runtime/src/rialto_messages.rs

```diff
@@ -40,6 +40,8 @@ pub const INITIAL_RIALTO_TO_MILLAU_CONVERSION_RATE: FixedU128 = FixedU128::from_
 parameter_types! {
 	/// Rialto to Millau conversion rate. Initially we treat both tokens as equal.
 	pub storage RialtoToMillauConversionRate: FixedU128 = INITIAL_RIALTO_TO_MILLAU_CONVERSION_RATE;
+	/// If `true`, all Rialto -> Millau messages are rejected.
+	pub storage RejectOutboundMessages: bool = false;
```

Comment:
Why not add this parameter directly to the pallet? This is not a bridge-specific parameter and as you mentioned it will need to be copy-pasted for every bridge instance.

We already have `IsHalted` in the pallet now. I think the two could either be merged in one `Config` object with two bool values or we could introduce `enum OperatingMode { Full, Halted, Inbound }`

The reason is that while the current solution reduces pallet-internal complexity it adds extra integration-level complexity and I feel that's already quite high.

Reply:
I'm considering this as 'our way' to ease upgrade process. Other may choose their own paths to solve that. Or, if they have non-upgradeable chains (like Rialto and Millau), they may just not use parameters like that. I would avoid adding any functionality that may need to be customized to the pallet itself - the pallet itself is not trivial && having potentially unused stuff there is undesirable.

Reply:
Actually - what's more important is that additional storage read. So since I don't want to make messages more expensive, I'll implement your option then